src/base/examples/images.html

```diff
@@ -21,7 +18,7 @@ <h3>Small Image</h3>
 </p>
 <p>
   <img
-    src="https://source.unsplash.com/320x180/daily?dogs"
+    src="https://s.mltshp.com/r/1ND3N?width=340&dpr=1"
```

Comment:
Let's use mltshp-cdn.com instead of s.mltshp.com ... the 's' subdomain triggers a view count.